src/main/java/unicash/model/transaction/Transaction.java

```diff
@@ -99,12 +99,13 @@ public boolean equals(Object other) {
         }
 
         Transaction otherTransaction = (Transaction) other;
-        return name.equals(otherTransaction.name)
-                && type.equals(otherTransaction.type)
-                && amount.equals(otherTransaction.amount)
-                && categories.equals(otherTransaction.categories)
-                && dateTime.equals(otherTransaction.dateTime)
-                && location.equals(otherTransaction.location);
+        boolean x1 = name.equals(otherTransaction.name);
+        boolean x2 = type.equals(otherTransaction.type);
+        boolean x3 = amount.equals(otherTransaction.amount);
+        boolean x4 = categories.equals(otherTransaction.categories);
+        boolean x5 = dateTime.equals(otherTransaction.dateTime);
+        boolean x6 = location.equals(otherTransaction.location);
```

Comment:
I think the previous form was fine, the booleans don't have a very descriptive name and trying to figure out a good name for them is probably not worth the effort

Reply:
Oh wait I did that for debugging, forgot to revert it. Thanks for catching